students/morgan/dnd_project/setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(
+	name='5e_encounter_builder',
+	author='Morgan H',
+	packages=['encounter_builder'],
+	# include_package_data=True,
+	# package_data={'CR_vals':['resource_files/cr_guide.csv'],
```

Comment:
nice to make a package!

figuring out the package data is a good idea -- take a look at the mailroom package solution.